__tests__/fixtures/index/run-custom-script-with-arguments/package.json

```diff
@@ -3,6 +3,6 @@
   "version": "1.0.0",
   "license": "UNLICENSED",
   "scripts": {
-    "custom-script": "print() { echo \"A message from custom script with args \"$@\"\"; }; print "
+    "custom-script": "node echo.js"
```

Comment:
Could use 
```
"custom-script": "node -e 'console.log(`A message from custom script with args ${process.argv[process.argv.length - 1]}`)'"
```

To avoid the need to have a separate .js file.

Reply:
That's what I did at first (I actually used `-p`) but Node 4 rejects additional arguments after the passed in script block. I used `--` to make it ignore but this was only added in Node 7 or 6 so it failed on Node 4 builds, hence this current, "ugly" solution :(